samples/otlptraceexport/src/app-http-proto-export.ts

```diff
@@ -0,0 +1,60 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import express, {Request, Response} from 'express';
+import {rollTheDice} from './dice';
+
+import {NodeSDK} from '@opentelemetry/sdk-node';
+import {OTLPTraceExporter} from '@opentelemetry/exporter-trace-otlp-proto';
+import {AuthClient, GoogleAuth} from 'google-auth-library';
+
+const PORT = parseInt(process.env.PORT || '8080');
+const app = express();
+
+async function getAuthenticatedClient(): Promise<AuthClient> {
+  const auth: GoogleAuth = new GoogleAuth({
```

Comment:
Seems to have been last PR
```suggestion
  const auth = new GoogleAuth({
```